### ISSUE TITLE
This project is still being worked on ？

### ISSUE THREAD

**Issue by 2563411574** [ardamavi/Game-Bot#33]
Hello, does this project no longer seem to be maintained?

**Comment by Stage4000** [comment]
yeah big dead for sure, let me know if you find any good alternatives.

**Comment by 2563411574** [comment]
Who is willing to continue to maintain it? I can provide training data

**Comment by Stage4000** [comment]
I sent out an email last night to someone who has recently updated this code base hopefully they get back to me. There are 2 recent updates to the code by 2 unconnected parties but I'm finding different issues on each. I'm pretty sure one of them changed python version because their requirements file is incompatible with 3.6 but they failed to update the readme to reflect if they're using 7,8,9.

**Comment by 2563411574** [comment]
Can it be unified?

**Comment by Stage4000** [comment]
Idek what that means man.

**Comment by FidgetySo** [comment]
@Stage4000 I think you are talking about me correct?

**Comment by Stage4000** [comment]
On the email part yes, but the one with the issues I discussed here is the other person who updated this code about 2 weeks ago.

**Comment by danielboos1** [comment]
https://github.com/FidgetySo/Game-Bot-1
